ams/server/src/main/java/com/netease/arctic/server/optimizing/maintainer/IcebergTableMaintainer.java

```diff
@@ -113,7 +115,12 @@ public void expireSnapshots(TableRuntime tableRuntime) {
         olderThanSnapshotNeedToExpire(tableRuntime), expireSnapshotNeedToExcludeFiles());
   }
 
-  public void expireSnapshots(long mustOlderThan) {
+  @Override
+  public void autoCreateTags(TableRuntime tableRuntime) {
+    new AutoCreateIcebergTagAction(table, LocalDateTime.now()).execute();
```

Comment:
Should the table be refreshed?
`table.refresh();`

Reply:
I think table will be refreshed by other service, I will check it later.

Reply:
Actually, the table here was just loaded.